game/tests.py

```
@@ -1338,3 +1339,101 @@ def test_chess_game_draws_on_insufficient_material_cpp_mocked(self):
         with mock.patch.object(game, '_call_engine', return_value="STATUS DRAW"):
             status = game.check_game_status()
             self.assertEqual(status, 'draw')
+
+
+class TimeoutResultRecordingTest(TestCase):
+    """A GameResult row must be persisted to the database when a player's clock expires."""
+
+    def _make_timed_out_session(self, timed_out_side='white', mode='pvp'):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_make_timed_out_session`

(ANN202)